packages/loader/container-loader/src/loader.ts

```diff
@@ -278,13 +292,24 @@ export type IDetachedBlobStorage = Pick<IDocumentStorageService, "createBlob" |
 export class Loader implements IHostLoader {
     private readonly containers = new Map<string, Promise<Container>>();
     public readonly services: ILoaderServices;
-    private readonly logger: ITelemetryLogger;
+    private readonly mc: MonitoringContext;
 
     constructor(loaderProps: ILoaderProps) {
         const scope = { ...loaderProps.scope as FluidObject<ILoader> };
         if (loaderProps.options?.provideScopeLoader !== false) {
             scope.ILoader = this;
         }
+
+        const subMc = mixinMonitoringContext(
+            DebugLogger.mixinDebugLogger("fluid:telemetry", loaderProps.logger, { all: { loaderId: uuid() } }),
+            ConfigProvider.create(
+                "Fluid",
+                [sessionStorageConfigProvider("FluidHot").value, // Global override for all configs
```

Comment:
@andre4i you move session storage here, but i think it needs to be in `ConfigProvider.create` the driver, loader, and runtime all move at different deployment speeds, so it's likely that the loader will be behind so just setting it here will lead to other layers not having it if this change isn't there yet